app/javascript/components/collections/landing/FacetBadges.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+
+class FacetBadges extends Component {
+  constructor(props) {
```

Comment:
Don't need this

Reply:
Since this does not handle state, can we convert this to a functional component as well?

Reply:
Sure thing.  Thinking about keeping this code relevant for the years to come, we should not really be using `class` components, unless absolutely necessary for some reason.   If a component needs to handle state, the `{ useState }` hook is pretty straightforward to wire in.